system/security-critical-code.md

```diff
@@ -12,22 +12,26 @@ redirect_from:
 Security-Critical Code in Qubes OS
 ==================================
 
-Below is a list of security-critical (AKA trusted) code in Qubes OS. A successful attack against any of those might allow to compromise the Qubes OS security. This code can be thought of as of a Trusted Computing Base (TCB) of Qubes OS. The goal of the project has been to minimize the amount of this trusted code to an absolute minimum. The size of the current TCB is of an order of hundreds thousands of lines of C code, which is several orders of magnitude less than in other OSes, such as Windows, Linux or Mac, where it is of orders of tens of millions of lines of C code.
+Below is a list of security-critical (AKA trusted) code in Qubes OS. 
+A successful attack against any of these portions of the codebase might allow an attacker to compromise the Qubes OS security. This code can be thought of as the Trusted Computing Base (TCB) of Qubes OS. 
+One of the major security goals of the project has been to minimize the amount of this trusted code to an absolute minimum. 
+The size of the current TCB is of an order of hundreds thousands of lines of C code, which is several orders of magnitude less than in other OSes, such as Windows, Linux or Mac, where it comprises tens of millions of lines of C code. 
+Keeping the TCB small reduces the operating system's attack surface and limits the points of entry for [scumbags like these](https://www.forbes.com/sites/andygreenberg/2012/03/21/meet-the-hackers-who-sell-spies-the-tools-to-crack-your-pc-and-get-paid-six-figure-fees/#4f6f605c1f74).
```

Comment:
I don't think the phrase "scumbags like these" or the quality of the article is consistent with the rest of the Qubes documentation.

Also, it appears that by the individuals referenced in the article and the text of the link, you are causing Qubes to endorse the position that the grugq (who is respected by many in the security community) is a scumbag.

Can we not? Please?